lib/server/readMetadata.js

```diff
@@ -149,28 +132,19 @@ function processMetadata(file) {
     metadata.title = metadata.id;
   }
 
-  if (languages.length === 1 && !siteConfig.useEnglishUrl) {
```

Comment:
So, `siteConfig.useEnglishUrl` is a config option (that is not documented, unfortunately - but I will document it if we agree), that allows one to override the default of `/docs/doc.html` with something like `/docs/en/doc.hmtl`.

Are you thinking we shouldn't allow this override any longer?

Reply:
my mistake. should keep this override. should we use this override, or just have a languages.js with only 'en' enabled. It is a separate question.

Reply:
My personal opinion is that we document and use the override to keep a clean separation between sites that have translations and sites that do not. If you have a `language.js` file, then translations will be enabled. If you want the `/docs/en` format, but no translations, you can use this override.

Seem reasonable?